frontend/app/components/ToolCard.tsx

```diff
@@ -30,32 +27,21 @@ export default function ToolCard (props: ToolCardProps) {
   const { tool } = props;
 
   const [learnMoreActive, setLearnMoreActive] = useState(false);
-
-  let logoImageProps: CardImageProps = {
-    component: 'img',
-    height: 150,
-    alt: tool.logo_image_alt_text ?? 'Logo for tool'
-  };
-  if (tool.logo_image !== null) {
-    logoImageProps = { image: tool.logo_image, ...logoImageProps };
-  }
-
-  let mainImageProps: CardImageProps = {
-    component: 'img',
-    height: 150,
-    alt: tool.main_image_alt_text ?? 'Image of tool in use'
-  };
-  if (tool.main_image !== null) {
-    mainImageProps = { image: tool.main_image, ...mainImageProps };
-  }
```

Comment:
I commented in an earlier PR that I didn't like how I did this. I found while working on this that `CardMedia` component was easier to use (and probably more readable) when I just used things inline. I forget why I did not do that earlier.